### DISCUSSION TITLE
Extending the map later on

### DISCUSSION THREAD

**Discussion by f-fritz** [Hypfer/Valetudo#1669] [Q&A / Support] [answered]
Hey

I could not find any information on that topic - maybe I used the wrong keywords during search:

I tried to extend the map of my Roborock S5 to include a small supply closet that I missed during the initial mapping process. The room does appear on the (already segmented) map view - but it has a dark blue color and no triangle and can therefore not be segmented or named or merged. Even if I reset all segmentations, the new area still has a different color and can't be edited.
Is there a way to extend a map later on other than discarding the entire map and start a new mapping process? Is this a problem with the Roborock firmware or Valetudo?

Thanks!

Firmware 002034
Valetudo 2022.11.0

**Comment by VossiVoss** [comment, marked as answer]
Try to perform a complete cleaning. The map should be enlarged automatically.  At least that's how it works on the DreamD9.

**Reply by f-fritz** [reply]
Thanks for the hint! You're right: When waiting for the full clean cycle to be completed, the newly discovered parts of the map change their color from blue to something else and can then be modified / labelled in the map editor as expected. 👍🏼
